packages/apps/frontera/src/store/Invoices/Invoice.store.ts

```diff
@@ -65,8 +65,12 @@ export class InvoiceStore extends Syncable<Invoice> {
     return this.root.settings.bankAccounts?.toArray();
   }
 
+  getId() {
+    return this.value.metadata.id;
+  }
+
   getChannelName() {
-    return 'Invoice';
+    return 'Invoices';
```

Comment:
is this intentional? This store is singular

Reply:
Yes. Organizations also has this approach. I've adjusted these 2 entities to use 1 group channel instead of having 1 channel for each obj.